AQA_Lab_Task/Program.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Driver;
+
+var driverGenerator = new DriverGenerator();
+var drivers = driverGenerator.GenerateDrivers();
+var vehicleGenerator = new VehicleGenerator(drivers);
+var vehicles = vehicleGenerator.GenerateVehicles();
```

Comment:
If you need to generate drivers and vehicles count as 2,5,10, you will go inside DriverGenerator and VehicleGenerator - modify count variable.
Create a variable and pass count through constructor here or better to pass in Generate methods.